src/read.c

```diff
@@ -33,9 +33,6 @@
 // }
 static const size_t VISUALSAMPLEENTRY_SIZE = 78;
 
-static const char xmpContentType[] = AVIF_CONTENT_TYPE_XMP;
-static const size_t xmpContentTypeSize = sizeof(xmpContentType);
```

Comment:
Yannis: It seems that we can use `AVIF_CONTENT_TYPE_XMP` and `sizeof(AVIF_CONTENT_TYPE_XMP)` directly. It doesn't seem necessary to copy `AVIF_CONTENT_TYPE_XMP` to a static variable (`xmpContentType`).

I hope I didn't miss something subtle...